src/python/ServerUtilities.py

```diff
@@ -750,6 +751,45 @@ def downloadFromS3(crabserver=None, filepath=None, objecttype=None, taskname=Non
                                         tarballname=tarballname, logger=logger)
     downloadFromS3ViaPSU(filepath=filepath, preSignedUrl=preSignedUrl, logger=logger)
 
+def checkS3Object(crabserver=None, objecttype=None, username=None, tarballname=None,
+                 logger=None):
+    """
+    Check if file exist in S3. Raise exception if wget is exit with non-zero.
+    Usually, you will see stderr with http response `404 Not Found` if file does not exists.
+    Note that presigned url from GetObject API could not use by HeadObject API.
+    Use 'head -c1000' to fetch few bytes instead, and need to wrap inside bash
```

Comment:
?? can you explain ? I have not found a use of head -c100 in the code.

Reply:
Sorry, it is the leftover from previous attempt to avoid extra code on REST side. 
It does not work. So switching to HeadObject api instead.